src/game/logic/ai/ai.h

```diff
@@ -58,6 +83,7 @@ class AISideBuildList : public MemoryPoolObject
     Utf8String m_name;
     BuildListInfo *m_buildListInfo;
     AISideBuildList *m_next;
+    friend class TAiData;
 };
 
 class TAiData : public SnapShot

```

Comment:
`TAiData` is intended name? Not `AIData`?

Reply:
This is correct.